### DISCUSSION TITLE
@formily/vue在CodeSandbox不支持吗？

### DISCUSSION THREAD

**Discussion by luoxuzhi** [alibaba/formily#2199] [Q&A 2.x] [answered]
查找了报错原因，实际是因为 @formily/vue 依赖了库 vue-demi，代码在本地是可以正常运行的。
依赖库版本如下：
    "@formily/core": "^2.0.0-rc.4",
    "@formily/element": "^2.0.0-rc.4",
    "@formily/reactive": "^2.0.0-rc.4",
    "@formily/reactive-vue": "^2.0.0-rc.4",
    "@formily/vue": "^2.0.0-rc.4",
    "@vue/composition-api": "^1.1.3",

附上 [地址](https://codesandbox.io/s/formconsumerdemo-8z8v3?file=/src/components/index.vue)

![image](https://user-images.githubusercontent.com/24772482/133913226-19515423-959b-4493-8cb9-6156fa183d83.png)

**Comment by muuyao** [comment, marked as answer]
请使用模板 https://codesandbox.io/s/github/Justineo/vue-demi-vue-2-template 因为 vue-demi 是基于 postinstall hook 来执行的，采用vue-cli 默认模板不会触发该hook
